ecosystem/sep-0006.md

```diff
@@ -844,6 +851,17 @@ An anchor should also indicate in the `/info` response if they support the `fee`
 * `enabled`: `true` if the endpoint is available.
 * `authentication_required`: `true` if client must be [authenticated](#authentication) before accessing the endpoint.
 
+#### Feature Flags
+
+The `feature_flags` object contains boolean values indicating whether or not specific features are supported by the anchor. If the object or specific feature is not present in the response, the default value described below may be assumed. This information enables wallets to adjust their behavior based on the feature set supported by the anchor.
+
+Name | Default | Description
+-----|---------|------------
+`account_creation` | `true` | Whether or not the anchor supports creating accounts for users requesting deposits.
+`claimable_balances` | `false` | Whether or not the anchor supports sending deposit funds as claimable balances. This is relevant for users of Stellar accounts without a trustline to the requested asset.
+
+The default values for the features listed above have been selected based on the ecosystem's current support. It is highly recommened to support all features enumerated for the best user experience.
```

Comment:
I don't think we need to say it's recommended, which has the spec offer opinion, rather we can focus on highlighting how to get the best use experience. 

```suggestion
The default values for the features listed above have been selected based on the ecosystem's current support. Supporting all features provided the best user experience.
```